spec/features/navigation/regular_user_navigation_spec.rb

```diff
@@ -3,18 +3,19 @@
 RSpec.describe "Registered User" do
   describe "User's View" do
     before(:each) do
-      @user = User.create(name: "Dee",
-                         street_address: "4233 Street",
-                         city: "Golden",
-                         state: "CO",
-                         zip: "80042",
-                         email: "deedee@gmail.com",
-                         password: "rainbows1908",
-                         role: 0)
+      @user = User.create(
+        name: "Dee",
+        street_address: "4233 Street",
+        city: "Golden",
+        state: "CO",
+        zip: "80042",
+        email: "deedee@gmail.com",
+        password: "rainbows1908",
+        role: 0
+      )
```

Comment:
I think this format is much more readable, I'll make sure to use this going forward